src/foam/lib/query/AfterGtParser.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.lib.query;
+
+import foam.lib.parse.Literal;
+import foam.lib.parse.PStream;
+import foam.lib.parse.Parser;
+import foam.lib.parse.ParserContext;
+import foam.lib.parse.Seq1;
+
+public class AfterGtParser extends foam.lib.parse.ProxyParser {
+
+  public AfterGtParser(Parser valueParser) {
+    setDelegate(new Seq1(1,
+                         new Literal(">"),
+                         valueParser
+                        ));
+  }
+
+  @Override
+  public PStream parse(PStream ps, ParserContext x) {
+    ps = super.parse ( ps, x );

```

Comment:
FOAM style is super.parse(ps, x); no extra spaces for method calls, only for control contstructs like "if" "while" etc.